tests/compute.py

```diff
@@ -427,3 +427,100 @@ def compute_image_3d_load_store(cfg: SceneCfg, show_dbg_points=False):
         group.add_children(get_debug_points(cfg, cuepoints))
 
     return group
+
+
+_IMAGE_CUBE_STORE_COMPUTE = """
+void main()
+{
+    ivec3 pos = ivec3(gl_LocalInvocationID.xyz);
+    vec4 color = vec4(0.0);
+    if (pos.z == 0) // right
+        color = vec4(1.0, 0.0, 0.0, 1.0);
+    else if (pos.z == 1) // left
+        color = vec4(0.0, 1.0, 0.0, 1.0);
+    else if (pos.z == 2) // top
+        color = vec4(0.0, 0.0, 1.0, 1.0);
+    else if (pos.z == 3) // bottom
+        color = vec4(1.0, 1.0, 0.0, 1.0);
+    else if (pos.z == 4) // back
+        color = vec4(0.0, 1.0, 1.0, 1.0);
+    else if (pos.z == 5) // front
+        color = vec4(1.0, 0.0, 1.0, 1.0);
+    imageStore(texture, pos, color);
+    imageStore(texture, pos, color);
+    imageStore(texture, pos, color);
+}
+"""
+
+_IMAGE_CUBE_VERT = """
+void main()
+{
+    ngl_out_pos = ngl_projection_matrix * ngl_modelview_matrix * vec4(ngl_position, 1.0);
+    var_uvcoord = ngl_position;
+}
+"""
+
+_IMAGE_CUBE_FRAG = """
+// Cube map texture selection is described in the OpenGL specification, see:
+// https://registry.khronos.org/OpenGL/specs/es/3.0/es_spec_3.0.pdf#table.3.21
+vec2 get_sample_cube_coord(vec3 v, out int face)
+{
+    vec3 v_abs = abs(v);
+    vec2 uv;  // sc,tc
+    float ma; // major axis
+    if(v_abs.z >= v_abs.x && v_abs.z >= v_abs.y) { // z major
+        uv = vec2(v.z < 0.0 ? -v.x : v.x, -v.y);
```

Comment:
I think you can use `sign(v.z) * v.x` instead of `v.z < 0.0 ? -v.x : v.x`, same below

Reply:
Done.